<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>StartGameSettingsState</class>
    <widget class="QWidget" name="StartGameSettingsState">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1097</width>
                <height>596</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2,5">
                    <property name="sizeConstraint">
                        <enum>QLayout::SetDefaultConstraint</enum>
                    </property>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Preferred</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Server IP Address:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="lineEdit">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Server Port:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="lineEdit_2">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_3">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Select Card Group:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="comboBox">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_4">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="pushButton">
                                    <property name="text">
                                        <string>Connect To Server</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="pushButton_2">
                                    <property name="text">
                                        <string>Exit</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer_5">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Preferred</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
